config/main.py

```diff
@@ -1468,8 +1483,12 @@ def rollback(ctx, checkpoint_name, dry_run, ignore_non_yang_tables, ignore_path,
        <checkpoint-name>: The checkpoint name, use `config list-checkpoints` command to see available checkpoints."""
     try:
         print_dry_run_message(dry_run)
-
-        GenericUpdater().rollback(checkpoint_name, verbose, dry_run, ignore_non_yang_tables, ignore_path)
+        asic_list = [multi_asic.DEFAULT_NAMESPACE]
+        if multi_asic.is_multi_asic():
+            asic_list.extend(multi_asic.get_namespace_list())
+        for asic in asic_list:
+            GenericUpdater(namespace=asic).rollback(checkpoint_name, verbose, dry_run,
```

Comment:
>rollback [](http://example.com/codeflow?start=43&length=8)

The rollback should not be at ASIC scope. We do not want to reach a status that some ASIC rolled back, but some are not.

Reply:
Also applicable to the checkpoint.